Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -260,6 +260,14 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+/** Set whether or not to log to stdout.
+ *
+ * @param enabled Whether or not stdout logging should be enabled.
+ *
+ * By default stdout logging is enabled.
+ */
+void kslog_setLogToStdout(bool enabled);
```

Comment:
Oh geeze. OK, forget all the BS I just spewed :P

I changed KSCrash awhile back such that the user no longer determines what the logfile name is on disk, because that's pretty much never useful. I changed it so that the logfile name and location is always the same, deterministic value so that it can add log files to reports, print previous logs, etc.

Since the logs are ALWAYS written to disk, and the user can determine whether to include them in reports or not, or print them after the fact, the only remaining annoyance is that it always prints log messages to STDOUT.

So what we actually need is a boolean flag telling it whether to also print to stdout or not (and default to FALSE). No more gating it on whether the report file path is NULL or not.

Sorry, been a long week!